archive/backend-backups/main_inmemory_backup_1753734039.py

```
@@ -0,0 +1,1692 @@
+    
+    return {
+        "employee": employee,
+        "property": property_obj,
+        "applicant_data": applicant_data
+    }
+
+@app.post("/onboard/{employee_id}/upload-document")
+async def upload_document(
+    employee_id: str,
+    document_type: str = Form(...),
+    file: UploadFile = File(...)
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)